Tasks/FtpUploadV2/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -21,8 +21,6 @@
   "loc.input.help.clean": "Delete the remote directory including its contents before uploading.",
   "loc.input.label.cleanContents": "Clear remote directory contents",
   "loc.input.help.cleanContents": "Recursively delete all contents of the remote directory before uploading.  The existing directory will not be deleted.  For better performance, consider using `Delete remote directory` instead.",
-  "loc.input.label.overwrite": "Overwrite",
```

Comment:
Why was this removed?

Reply:
Because ftp doesn't offer this functionality, and to make this work, we are making extra calls to server, this gets expensive really soon as we need to make extra calls on each file, and will contribute to server dropping the connection.

Reply:
Do other popular FTP clients offer this?

Reply:
I checked a few popular libraries, none of them offer "STOU" (store unique, it generates a new filename if the current file exists) call.  "STOU" is an optional cmd ftp server can support.